### ISSUE TITLE
UnicodeDecodeError: 'cp950' codec can't decode byte 0xcf in position 15472: illegal multibyte sequence

### ISSUE THREAD

**Issue by acyang** [LFPy/LFPy#452]
**Describe the bug**
Encountered the following error when install lfpy via pip install.
UnicodeDecodeError: 'cp950' codec can't decode byte 0xcf in position 15472: illegal multibyte sequence

**To Reproduce**
1. pip install lfpy

**Expected behavior**
It supposed to be installed without any error.

**Screenshots**
![圖片](https://github.com/LFPy/LFPy/assets/2888344/7c23014f-2a62-4722-9868-860c850a9b14)

**Desktop/Environment (please complete the following information):**
 - OS: window10
 - Shell: cmd
 - Python-Version: python3.10
 - LFPy version: LFPy-2.3
 - Installation: pip

**Additional context**
setup.py line 41
```
with open('README.md', 'rt') as file:
with open('README.md', 'rt', encoding="UTF-8") as file:
```
will fix this issue

**Comment by espenhgn** [comment]
Great if you could provide a fix. LFPy is not routinely tested/developed on Windows so such issues may go unnoticed :)

**Comment by torbjone** [comment]
Thank you for pointing this out, it should now be solved by PR #458. Please let us know it there are still problems :-)
